Packs/CrowdStrikeFalconX/Integrations/CrowdStrikeFalconX/CrowdStrikeFalconX.py

```diff
@@ -330,7 +336,7 @@ def parse_outputs(
     """
     if api_res.get("errors"):
         # if there is an error in the api result, return only the error
-        return api_res.get("errors")
+        return {"errors": api_res.get("errors")[0]}
```

Comment:
what happens if there is more than one error in the list?
let's do a utility func for this and make sure we unit testts all the options:
no "erorrs", errors is a string/list with 1 item/multiple items/empty list